feat(guide): include the different hdr formats in the quality profiles (#1623)
TRaSH-Guides/Guides: docs/Radarr/radarr-setup-quality-profiles.md

## TRaSH Quality Profiles

> All the Quality Profiles here can be combined into one Quality Profile if you want to be able to upgrade, for example, from 1080p to 4k/2160p.

Reviewer: ```suggestion
The following Quality Profiles can be combined into a single Quality Profile if you, for example, want to be able to upgrade from 1080p to 4K/2160p.
```

------

## Custom Format Groups

> The following custom format groups should be used in combination with the above quality profiles. Users will need to make a choice as to whether they are included, or which option to go for.

Reviewer: ```suggestion
The following custom format groups should be combined with the Quality Profiles above. Users will need to make a choice as to which options and custom formats they prefer.
```

### Advanced Audio

- You have a audio setup that supports all HD Audio (TrueHD, DTS-X, DTS-HD, etc).
- You have a media player (hardware devices) that can passthrough HD audio.
- You don't have a AppleTV (Being it can't passthrough untouched HD audio).

> *The reason that lossy Atmos is scored higher than lossless DTS-HD MA is because I prefer having the object metadata (Atmos) over lossless audio.*

Reviewer: ```suggestion
*The reason that I score lossy Atmos higher than lossless DTS-HD MA is that I prefer having the object metadata (Atmos) over lossless audio.*
```

    Use x265 only for 4k releases and the [{{ radarr['cf']['x265-hd']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#x265-hd){:target="_blank" rel="noopener noreferrer"} makes sure you still get the x265 releases.

> ### Why am i getting purple or green colors

Reviewer: ```suggestion
### Why am I getting purple or green colors?
```
